### ISSUE TITLE
Axes labels offset from points in cases with small amount of data

### ISSUE THREAD

**Issue by sharlagelfand** [microsoft/datamations#91]
@giorgi-ghviniashvili I noticed a bug where the axes labels and points don't match up when there is a small amount of data, e.g. 10 points:

```r
library(dplyr)
library(datamations)

small_salary_sample <- small_salary %>%
  group_by(Degree) %>%
  sample_n(5) %>%
  ungroup()

"small_salary_sample %>% group_by(Degree)" %>% datamation_sanddance()
```

The second frame looks like this:

<img width="660" alt="Screen Shot 2021-07-16 at 2 51 20 PM" src="https://user-images.githubusercontent.com/15895337/125995489-77f5a566-afce-4f61-85e7-0624a7c7c361.png">

It seems to level off and look okay around 50 points, but any smaller is off. You can play around with different group numbers by editing the `n`s in the [specs](https://github.com/microsoft/datamations/blob/small-sample-bug/sandbox/small_data/small_data_specs.json). Thanks!

**Comment by giorgi-ghviniashvili** [comment]
@sharlagelfand good point, pushed the fix for that.

**Comment by sharlagelfand** [comment]
Closed by #92.
